semantic_conventions/metrics/rpc-metrics.yaml

```diff
@@ -0,0 +1,116 @@
+groups:
+  # TODO: Should we list the attributes on each metric
+  # OR leave a single table like it is today? Since all attributes are applied
+  # to all metrics, the repetition of them bloats the page
+  - id: attributes.metrics.rpc
+    type: attribute_group
```

Comment:
If we are OK in leaving the attributes in their own table, I will then remove this `attribute_group` here.

Reply:
I think we should probably have the attributes per-metric here. I *think* client vs. server attributes are actually different, I'm not sure where the language around that got confused.